Makefile

```diff
@@ -227,6 +227,8 @@ PHONY_MODULES := $(foreach x,$(MODULE_NAMES),build/modules/$(x).owl)
 build/terms-report.csv: build/obi_merged.owl src/sparql/terms-report.rq | build
 	$(ROBOT) query --input $< --select $(word 2,$^) $@
 
+# Always get the most recent version of OBI - in case build directory has not been cleaned
+.PHONY: build/obi-previous-release.owl
```

Comment:
I'm happy to merge this PR now, but please work toward a better alternative to this: files should never be phony; this is a "bad code smell" indicating that the overall build flow should change in some way.

Reply:
Here's an alternative:
```
.PHONY: candidate
candidate:
	$(eval REMOTE := $(shell git remote -v | grep "https://github.com/obi-ontology/obi.git" | head -1 | cut -f 1))
	rm -rf build/obi-previous-release.owl
	make all build/new-entities.txt
	git checkout -b $(TODAY)2
	...
```

The cleanest would just be:
```
.PHONY: candidate
candidate: clean all build/new-entities.txt
	...
```
... but this rebuilds `obi.owl` completely and this is a waste of time if you've already built it, since the HermiT reasoning takes so long... My alternative above only builds it if it hasn't yet been built, but still ensures that we have the latest previous release.

Reply:
Yes, that does too much work. I think recursive `make` is acceptable in this case:

```
.PHONY: candidate
 candidate: obi.owl views
 	make -B build/new-entities.txt
 	$(eval REMOTE := $(shell git remote -v | grep "https://github.com/obi-ontology/obi.git" | head -1 | cut -f 1))
 	git checkout -b $(TODAY)
 	git add -u
 	git commit -m "$(TODAY) release candidate"
 	git push -u $(REMOTE) $(TODAY)
 	gh pr create \
 	--title "$(TODAY) release candidate" \
 	--body "$$(cat build/new-entities.txt)" \
 	--repo obi-ontology/obi
```

Or maybe that first line should be `make -B build/current-entities.tsv build/new-entities.txt`

For this Makefile I think it's safe to hardcode `origin` rather than do this dance to figure out `REMOTE`. The current code doesn't work on my system anyway, since I have it configure like this:

```
$ git remote -v
origin  git@github.com:obi-ontology/obi.git (fetch)
origin  git@github.com:obi-ontology/obi.git (push)
```

Reply:
I'm happy for you to merge this as it is.